cl_sii/cte/f29/parse_datos_obj.py

```diff
@@ -22,27 +22,50 @@
 CTE_F29_DATOS_OBJ_SCHEMA = read_json_schema(_CTE_F29_DATOS_OBJ_SCHEMA_PATH)
 
 
-def parse_sii_cte_f29_datos_obj(datos_obj: SiiCteF29DatosObjType) -> CteForm29:
+def parse_sii_cte_f29_datos_obj(
+    datos_obj: SiiCteF29DatosObjType,
+    validate_schema: Optional[Callable[[SiiCteF29DatosObjType], None]] = None,
+    deserialize_campo: Optional[Callable[[object, str], object]] = None,
```

Comment:
Perhaps `campo_deserializer` would be more appropriate?

Reply:
At first I used `campo_deserializer`, but given that it's a function, I thought that `deserialize_campo(value, datos_obj_tipos[code])` looked better than `campo_deserializer(value, datos_obj_tipos[code])`.

What do you think?

Reply:
Whatever we decide it should be consistent with the decision for the comment above